controller/controller.go

```diff
@@ -122,3 +115,44 @@ func Process(version string) {
 	log.Debug().Str("Parsed API", parsedAPI.String()).Msg("")
 	HandleParsedFile(parsedAPI, o, dir)
 }
+
+func processEnvironmentVariables(o *lama2cmd.Opts, directory string) {
+	envMap, err := preprocess.GetL2EnvVariables(directory)
+	if err != nil {
+		log.Error().Str("Type", "Preprocess").Msg(err.Error())
+		os.Exit(0)
+	}
+	if o.Env == "" { // -e=''
```

Comment:
The order of these checks must be simplified:

1. Is it an `-e` invocation? ( != unset)
    1. is it empty? handle
    2. is it not empty? handle
2. Not an `-e` continue.

Reply:
Also if it is a `-e` request, for efficiency sake, once printing is done, we should simply exit the app, right? We are not doing that now, continuing processing.

Essentially `-e` is a editor extension support command; we don't expect a normal user to poke into it.

Reply:
Got it